docs/src/pages/components/flex/FlexPropControls.tsx

```diff
@@ -0,0 +1,121 @@
+import { FlexProps, TextField, SelectField, Flex } from '@aws-amplify/ui-react';
+import React from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```
We are now consistent on using namespace import